love-letter-app/src/app/ui/usablesvgs.tsx

```diff
@@ -0,0 +1,36 @@
+// here we will create an array with all svg names and then map over them to render them in the page
+'use client';
+import Image from "next/image";
+import { useState, useEffect } from "react";
+
+export default function UsableSVGs(props: any) {
+    const svgNames = ["envelope.svg", "envelope1.svg"];
+
+
+    const handleSVGClick = (e: any) => {
+        const svgs = document.getElementsByClassName("selected-svg");
+        for (let i = 0; i < svgs.length; i++) {
+            svgs[i].className = "selectable-svg";
+        }
+        props.setSelectedSVG(e.target.alt);
+        e.target.className = "selected-svg";
+        console.log(e.target);
```

Comment:
I see that this function toggles the class of the selected and non selected.
Look into the possibility of using something like this:

<Image className={`${isSelected ? 'selectable-svg' : 'selected-svg'}`} ...

Reply:
addressed